File: coding.rb
# Menulis ke file
File.open("data.txt", "w") do |file|
    file.puts "Ini adalah contoh teks yang ditulis ke file."
    file.puts "Baris kedua dari file."
  end
  
  # Membaca dari file
  File.open("data.txt", "r") do |file|
    file.each_line { |line| puts line }
  end

  
  def cek_prima(n)
    return false if n <= 1
    (2..Math.sqrt(n).to_i).none? { |i| n % i == 0 }
  end
  
  print "Masukkan angka untuk diperiksa keprimaan: "
  number = gets.to_i
  if cek_prima(number)
    puts "#{number} adalah bilangan prima."
  else
    puts "#{number} bukan bilangan prima."
  end
  

File: coding.kt

import kotlin.math.PI

fun luasLingkaran(radius: Double): Double {
    return PI * radius * radius
}

fun main() {
    print("Masukkan jari-jari lingkaran: ")
    val radius = readLine()!!.toDouble()
    println("Luas lingkaran dengan jari-jari $radius adalah ${luasLingkaran(radius)}")
}

import java.io.File

fun main() {
    val fileName = "data.txt"

    // Menulis ke file
    File(fileName).printWriter().use { out ->
        out.println("Ini adalah contoh teks yang ditulis ke file.")
        out.println("Baris kedua dari file.")
    }

    // Membaca dari file
    File(fileName).forEachLine { println(it) }
}
